tests/comparisons/Create/TestCreate.php

```diff
@@ -0,0 +1,16 @@
+<?php
+use Migrations\AbstractMigration;
+
+class TestCreate extends AbstractMigration
```

Comment:
I'll just ignore this comment, the `comparisons` folder will be ignored in the stickler file from #301